File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// TempMailApiByBoomlify SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class TempMailApiByBoomlifyMakeContext
{
    public static function call(array $ctxmap, ?TempMailApiByBoomlifyContext $basectx): TempMailApiByBoomlifyContext
    {
        return new TempMailApiByBoomlifyContext($ctxmap, $basectx);
    }
}

File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/temp-mail-api-by-boomlify-sdk/go/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}
